src/app.ts

```diff
@@ -339,9 +327,6 @@ export async function getApp(
 
   try {
     await app.ready()
-    if (!isTest() && configOverrides.healthchecksEnabled !== false) {
-      await runAllHealthchecks(app)
```

Comment:
this is redundant, liveness probe on a startup is going to kill service in case healthchecks don't pass anyway